models/instrument.py

```diff
@@ -1208,7 +1735,10 @@ def download_exposures( self, outdir=".", indexes=None, clobber=False, existing_
 
         Returns
         -------
-        A list of pathlib.Path for the files that were downloaded.
+        A list of dictionaries, each element of which is { prod_type: pathlib.Path }
+        prod_type will generally only be "exposure", but if there are ancillary
+        exposures, there may be others.  E.g., when downloading reduced DECam
+        images from NOIRlab, prod_type may be all of 'exposure', 'weight', 'dqmask'.
```

Comment:
Probably putting them in exposure extensions.  But, I'm thinking for DECam reduced images, we may not want to save the exposures at all, since the processing from exposure to image is simply an extraction.  It's a lot of gratuitous extra disk space to save the exposures.  That'd require thought, though.